ivy_tests/test_ivy/test_functional/test_core/test_statistical.py

```diff
@@ -347,7 +347,6 @@ def test_cumsum(
     )
 
 
-# cumprod
```

Comment:
Why did we remove this?

Reply:
I must have accidentally removed this comment from the top of  the cumprod test function. Let me fix it and also add a `# cummin` above my implementation. apologies